### PR TITLE
Implement editp command

### LINKED ISSUE #166: Add editP command

### PR DESCRIPTION
**Pull request by aslam341**
Closes #166.

Changes made:

1. Add `EditProjectCommand`
2. Change `EditCommand` -> `EditEmployeeCommand`
3. Change command word for `EditEmployeeCommand` from `edit` to `editE`

New Command: `editP`
Command usage: `editP INDEX [n/NAME] [priority/PRIORITY] [d/DEADLINE]`, and at least one field out of the three must be present.
Example: 
1. `editP 1 n/Infinity` 
2. `editP 1 d/29-10-2023 priority/low`

Yet to add in UG.

### CODE REVIEW

**Review by antonTan96 — APPROVED** [review]
LGTM!

File: src/main/java/seedu/address/logic/commands/EditProjectCommand.java [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+    public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
+    public static final String MESSAGE_DUPLICATE_PROJECT = "This project already exists in the TaskHub.";
+
+    public final Index index;
+    public final EditProjectDescriptor editProjectDescriptor;
+
+    /**
+     * @param index of the project in the filtered project list to edit
+     * @param editProjectDescriptor details to edit the project with
+     */
```
Comment (Chandan8186) [inline src/main/java/seedu/address/logic/commands/EditProjectCommand.java:51]: I think you may have forgotten the Javadoc description?

**Review by Chandan8186 — CHANGES_REQUESTED** [review]
LGTM! Just a minor comment regarding Javadoc.

File: src/main/java/seedu/address/logic/commands/EditProjectCommand.java (hunk 1 above)
Comment (aslam341) [reply src/main/java/seedu/address/logic/commands/EditProjectCommand.java:51]: There is nothing noteworthy to say here, and this is how it is written in `EditEmployeeCommand.java` as well.

**Review by Chandan8186 — APPROVED** [review]
LGTM

**Pull request merged**